config/import/content/homepage.php

```diff
@@ -0,0 +1,263 @@
+<!-- /wp:columns -->
+
+<!-- wp:paragraph -->
+<p></p>
+<!-- /wp:paragraph --></div></div></div>
+<!-- /wp:atomic-blocks/ab-container -->
+
+<!-- wp:atomic-blocks/ab-spacer -->
+<div style="color:#ddd" class="wp-block-atomic-blocks-ab-spacer ab-block-spacer ab-divider-solid ab-divider-size-1"><hr style="height:30px"/></div>
+<!-- /wp:atomic-blocks/ab-spacer -->
+
+<!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide has-2-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:atomic-blocks/ab-accordion {"accordionOpen":true} -->
+<div class="wp-block-atomic-blocks-ab-accordion ab-block-accordion ab-font-size-18"><details open><summary class="ab-accordion-title">Can I phone or email you?</summary><div class="ab-accordion-text"><!-- wp:paragraph -->
+<p>You can reach us on our <a href="/contact">contact page</a>, by emailing test@example.com, or by calling 123456.</p>
+<!-- /wp:paragraph --></div></details></div>
+<!-- /wp:atomic-blocks/ab-accordion -->
+
+<!-- wp:atomic-blocks/ab-accordion -->
+<div class="wp-block-atomic-blocks-ab-accordion ab-block-accordion ab-font-size-18"><details><summary class="ab-accordion-title">How long have you been in business?</summary><div class="ab-accordion-text"><!-- wp:paragraph -->
+<p>Our team has a combined experience of over 60 years.</p>
+<!-- /wp:paragraph --></div></details></div>
+<!-- /wp:atomic-blocks/ab-accordion -->
+
+<!-- wp:atomic-blocks/ab-accordion -->
+<div class="wp-block-atomic-blocks-ab-accordion ab-block-accordion ab-font-size-18"><details><summary class="ab-accordion-title">What is your returns policy?</summary><div class="ab-accordion-text"><!-- wp:paragraph -->
+<p>We offer a 30-day returns policy for all unused items. For more information, please view our terms of service.</p>
+<!-- /wp:paragraph --></div></details></div>
+<!-- /wp:atomic-blocks/ab-accordion --></div>
+<!-- /wp:column -->
+
+<!-- wp:column -->
+<div class="wp-block-column"><!-- wp:image {"id":308} -->
+<figure class="wp-block-image"><img src="$genesis_homepage_faq_image_url" alt="" class="wp-image-308"/><figcaption>Photo by <a href="https://unsplash.com/photos/rjhvXX7JBPc">Haley Powers</a> on <a href="https://unsplash.com/">Unsplash</a></figcaption></figure>
+<!-- /wp:image --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:columns {"columns":3,"align":"wide"} -->
+<div class="wp-block-columns alignwide has-3-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:heading {"level":4} -->
+<h4>About us</h4>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>Describe yourself or your business in a short sentence or two.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column -->
+
+<!-- wp:column -->
+<div class="wp-block-column"><!-- wp:heading {"level":4} -->
+<h4>Share this page</h4>
+<!-- /wp:heading -->
+
+<!-- wp:atomic-blocks/ab-sharing {"google":false,"linkedin":true,"email":true,"shareButtonStyle":"ab-share-icon-only"} /--></div>
```

Comment:
@nickcernis the AB Sharing block notice can be fixed by adding `"shareAlignment":"center"` to this line. I think this is a bug in AB, but it'll fix it here for us now.

Reply:
Edited by above comment to say `"shareAlignment":"left"`, I accidentally pasted `center` from a test I was doing.

Reply:
Done — nice catch and thanks, @mindctrl!